modal/functions.py

```diff
@@ -235,13 +235,14 @@ def __init__(self, stub: api_grpc.ModalClientStub, function_call_id: str, client
         self.function_call_id = function_call_id  # TODO: remove and use only input_id
 
     @staticmethod
-    async def create(function_id: str, args, kwargs, client: _Client) -> "_Invocation":
+    async def create(function_id: str, nowait: bool, args, kwargs, client: _Client) -> "_Invocation":
```

Comment:
nit: `nowait` isn't immediately obvious what effects it has - maybe `set_parent` or something? Is this related to whether we should kill child calls when the parent is terminated or not?

Reply:
+1